src/xdocs/config_system_properties.xml

```diff
@@ -74,5 +74,31 @@
       </p>
     </section>
 
+    <section name="Property chaining support">
+      <p>
+        Checkstyle supports property expansion within property definitions, also
+        known as property chaining.  This feature allows you to define properties
+        using other properties.  For example:
```

Comment:
1. `.  For`
2. `chaining.  This`
double space

Reply:
Bad habit, fixed.